src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml

```diff
@@ -235,8 +235,8 @@ let setup_daemon logger =
   and work_selection_method_flag =
     flag "--work-selection" ~aliases:[ "work-selection" ]
       ~doc:
-        "seq|rand Choose work sequentially (seq) or randomly (rand) (default: \
-         rand)"
+        "seq|rand|offset Choose work sequentially (seq), randomly (rand), or \
```

Comment:
how do you feel about `roffset` or `rseq`?

Reply:
replaced with `roffset`